js/stats.js

```diff
@@ -0,0 +1,86 @@
+var authorArray = [];
```

Comment:
suggestion to declare a stats object (```var stats = {}```) so that you can attach all these vars to that object instead of having to anonymously declare each one.